src/client/Microsoft.Identity.Client/TelemetryCore/OpenTelemetry/IOtelInstrumentation.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Identity.Client.Core;
+
+namespace Microsoft.Identity.Client.TelemetryCore.OpenTelemetry
+{
+    internal interface IOtelInstrumentation
+    {
+        internal void LogActivity(Dictionary<string, object> tags);
+        internal void LogActivityStatus(bool success);
+        internal void StopActivity();
+        internal void LogSuccessMetrics(
+            string platform,
+            AuthenticationResultMetadata authResultMetadata,
+            string apiId,
+            string cacheLevel,
+            ILoggerAdapter logger, 
+            long totalDurationInUs);
```

Comment:
nit: tricky one, we can simply call this `totalDurationInMs`

Reply:
It is microseconds(us). I will add a comment here.

Reply:
The totalDurationInMs is present in authResultMetadata. We needed microseconds to record duration in case of L1 cache